### DISCUSSION TITLE
Video file permissions

### DISCUSSION THREAD

**Discussion by ana-rita-goncalves** [Motion-Project/motion#1767] [Q&A]
Hello

Is there a way of generating video outputs files that are not protected?

![Screenshot from 2024-04-14 20-07-19](https://github.com/Motion-Project/motion/assets/153758594/0ea99fed-3352-4eca-955d-2a59a7a196c1)

Thank you in advance!

**Comment by tosiara** [comment]
Which user is motion running? Looks like it is not the same user as viewing.

**Comment by RWL-69** [comment]
That is not motion configuration but a system level setting.  Look at different chmod commands to ensure that the directory that motion uses is accessible to all the users that you want to have access.  You also have to ensure that the users are in the correct group to access files.

Example.  My camera directory is set like this.
drwxrwsrwx+ 13 motion   video    32813056 Dec 31 12:18 cameras

User motion is in group video so all videos are part of the group video.

Since the directory is part of the group video, any user that is in the group video now can access and change files that have the correct permissions.

It took me some time to get this set up on my machine with SELinux permissions and to work over NFS.
